src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -167,4 +169,22 @@ public void linkDefendantToClaim(Long claimId, String defendantId) {
 
         claimRepository.linkDefendant(claim.getId(), defendantId);
     }
+
+    public void linkLetterHolder(final Long claimId, final String userId) {
+        claimRepository.linkLetterHolder(claimId, userId);
+    }
+
+    public void linkSealedClaimDocumentManagementPath(final Long claimId,
+                                                      final String sealedClaimDocumentManagementSelfPath) {
+        claimRepository.link(claimId, sealedClaimDocumentManagementSelfPath);
+    }
+
+    void saveCountyCourtJudgment(final long claimId, final CountyCourtJudgment countyCourtJudgment) {
+        claimRepository.saveCountyCourtJudgment(claimId, jsonMapper.toJson(countyCourtJudgment));
+    }
+
+    void saveDefendantResponse(final long claimId, final String defendantId, final String defendantEmail,
+                               final ResponseData responseData) {
+        claimRepository.saveDefendantResponse(claimId, defendantId, defendantEmail, jsonMapper.toJson(responseData));
```

Comment:
All these methods added here just forward the call to repository, can't see much value in that...

Reply:
I agree, however using claim repository directly didn't look good to me. All services should connect other through services

Reply:
Can you elaborate why using the repository directly is a problem?

Reply:
I'm guessing it's because you're thinking about services and repositories as application layers. This is a common misconception. Using them in such manner does not add much value and usually leads writing unneeded code just for the sake of it.

A _Service_ is just a piece of logic which does not belong to any particular model. This is according to [Domain Driven Design](https://www.amazon.co.uk/Domain-Driven-Design-Tackling-Complexity-Software/dp/0321125215), the book from which all these concepts (_Service_, _Repository_, etc.) actually came from.

Reply:
it was just a thought, it cleaner. we would be having other repositories like documentFiles, they should be accessed through their service. I agree as they are component, can be accessed from any where. no harm as such.